changelog/62482.fixed

```diff
@@ -0,0 +1 @@
+Update sanitizing masking for Salt SSH to include additional password like strings.
```

Comment:
I just noticed this changelog has a different number as https://github.com/saltstack/salt/pull/62483 this would mean in the 3006 release we would be adding this as fixed even though it was fixed in 3005. We can either rename this filename to 62483 and the merge forward will capture that the file was deleted. or just remove this changelog.